docs/SelectMulti.stories.ts

```diff
@@ -32,12 +34,14 @@ export const Primary = Template.bind({});
 Primary.args = {
 	label: 'My SelectMulti',
 	options: longDefaultListOfOptions,
+	labelField: 'label',
```

Comment:
Hi @Felixclement - thanks for the contribution! And thank you also for making sure the new props are added to Storybook for interaction there. It's great to be able to test this out easily, and everything's looking good to me.
https://deploy-preview-53--vue-accessible-selects.netlify.app/

Some notes:

1. Given that this is increasing the surface layer of the API for the components (i.e. more props), we'll want to carefully review. We were aware that we'd want something like these props to enable custom object shapes, so this is a great solution to that problem, and we'll definitely work towards merging
2. We'll also want to review carefully to make sure all accessibility-related features that were previously relying on only one value field, or only one label field, are still working correctly - everything there looks good so far too
3. Finally, you caught us before we have specs in place 😶 Props for being early! We're gonna see if we can move up our spec-writing ticket to get those in place this week or next, so that we can verify specs pass for this PR, but also add in something like "given custom options objects, fields still behave correctly when `valueField` and `labelField` are used"

So we may be just a bit slow in merging this, but we're absolutely working towards it, and thanks for your contribution!

Reply:
Great, thanks @jackkoppa for the very detailed feedback